src/signalrclient/request_sender.cpp

```diff
@@ -36,9 +37,9 @@ namespace signalr
         }
 
         pplx::task<void> start(web_request_factory& request_factory, const web::uri &base_url, transport_type transport,
-            const utility::string_t& connection_token, const utility::string_t &query_string)
+            const utility::string_t& connection_data, const utility::string_t& connection_token, const utility::string_t &query_string)

```

Comment:
I fixed this in a subsequent PR. I found it because an end to end scenario did not work because of this. I might be able to test this indirectly via web_request_sender.